docs/whats-new/staging-release-notes.md

```diff
@@ -1,5 +1,26 @@
 # Staging release notes
 
+## Staging release 15 September 2022
```

Comment:
hmm.. do we need to mention the updated ToU and PP ?

Reply:
yeah, need to..I am checking with Denise on what changes to be highlighted to the user.